assets/app/view/game/tranches.rb

```diff
@@ -0,0 +1,83 @@
+# frozen_string_literal: true
+
+require 'lib/settings'
+
+module View
```

Comment:
what does this look like and what is it for

Reply:
This game (like 1835) has tranches, only certain number of companies can open until all companies in the current tranch are either sold out or operated. This displays the tranch chart. We added it to the spreadsheet and stock round views. I can screenshot for you later.

Reply:
![tranches](https://user-images.githubusercontent.com/40592/156730255-51be034e-81c1-489f-a25a-51b6ae7be678.png)

Reply:
I used the status field to show this for 1860, 1825, and 18Carolinas.

Reply:
Designer would rather a tranch chart be viewable that try to put this information on each company. Makes it very clear what companies are taking up what tranches and if their are open options. It's only show during the stock rounds and spreadsheet view if the game is using tranches. Right now in 1871 we clean up the entities list during stock rounds to not include all of the companies that can't act / can't open. I'll gladly argue that adding a new module for this is worth it for this game and doesn't muddy up any shared game code.

Reply:
let's just go with status fields and perhaps the custom html chart that steve added for 21moon?